Gruppe.A1.End-to-End_Encryption_mit_IPv6_und_integriertem_IPSec.ThushjandanPonnudurai.PascalBaumann.Termpaper.tex

```diff
@@ -508,12 +508,39 @@ \section{Experiment results}
 IPsec support is very well implemented in Microsoft Windows 10, with a variety of configuration options and a comprehensive user interface. This led to a very uncomplicated setup of client ws01. Linux on the other hand (or in our case Ubuntu) ships with kernel support since 2.6, but this is only the kernel side. User mode configuration of tunnels need additional software. StrongSwan, in our case, has a logical structure of its files, but the usability and simplicity of configuration is lacking. One can argue that security should not be a simple matter, but Microsoft demonstrates that simplicity and security are not mutually exclusive. 
 
 \section{Further improvements}
-\subsection{Automation}
-% Tush : Automatisierung CA, Deployment, Evtl einlesen native IPsec implementation Linux feasibility
+After the practical experiment we reflected how an automation of an IPsec tunnel configuration process in a heterogeneous IPv6-only environment could look like. 
+According to chapter \ref{sec:IPv6IPsec} it should be possible to encrypt easily the traffic between two hosts, because IPsec is a component of IPv6 protocol.
+
+Firstly we have to automate the certificate enrollment process. Unfortunately it is not easily possible to realize this task with OpenSSL as CA. 
+Therefore Microsoft Active Directory Certificate Services (ADCS) will be a better choice than OpenSSL. 
+One of the main reasons is that ADCS can be perfectly integrated in an Active Directory environment. Certificate Autoenrollment can be easily configured with group policies for Windows computers. 
+Another reason for ADCS is that it supports the Simple Certificate Enrollment Protocol (SCEP).
+With SCEP it is possible to enroll automatically certificates for network devices and also for Linux machines. 
```

Comment:
[...] to _automatically roll out_ certificates for _any_ network device, _including_ machines _running a linux distribution_.